core/services/src/main/java/org/exoplatform/services/attachments/utils/Utils.java

```diff
@@ -86,8 +87,8 @@ public static String getCurrentWorkspace(RepositoryService repositoryService) th
     return repositoryService.getCurrentRepository().getConfiguration().getDefaultWorkspaceName();
   }
 
-  public static boolean isQuarantinedItem(Session systemSession, String attachmentId) throws RepositoryException {
-    Node attachmentNode = systemSession.getNodeByUUID(attachmentId);
+  public static boolean isQuarantinedItem(ExtendedSession systemSession, String attachmentId) throws RepositoryException {
```

Comment:
it should be better to keep Session type for systemSession param then cast it when it will be used